CONTRIBUTING.md

```diff
@@ -19,11 +19,31 @@ Before you create a new Issue:
 	* Also check that your code is formatted properly with spaces (hint: Use [.editorconfig](http://editorconfig.org/))
 * Keep git commit messages clear and appropriate
 	* If possible, please "squash" your commits to as few commits as possible/reasonable such as one commit for implementation, one for tests, and one for documentation before finally squashing to one commit when getting the LGTM from a collaborator.
+  * Follow the commit message guidelines below.
 * Make Sure you have added any tests necessary to test your code.
 	* Run __all__ the tests to ensure nothing else was accidently broken.
 	* Don't rely on the existing tests to see if you've broken code elsewhere; test the changes you made in a browser too!
 * Update the Documentation to go along with any changes in functionality / improvements in a separate pull request against the gh-pages branch.
 
+## Commit Message Guidelines
+
+* Format:
+  * `<type>(<scope>): <subject>`
+* Types: 
+  * feat - Features, Enhancements, and overall Improvements
+  * fix - Fixes, Bugs, HotFixs, etc...
+  * doc - Changes to the Documentation and doesn't actually touch any code.
+* Scope:
+  * The scope should be where the change took place.
+  * Examples: users, core, config, articles
+* Subject:
+  * The subject line should be clear and consice as to what is being accomplished in the commit.
+* General Rules:
+  * No Line in the Commit message can be longer than 100 characters.
```

Comment:
Why 100, if we do 80 in the project, why not 80 here as well? Is there a good reason not to?